Looking in indexes: http://pypi-cache/index, https://pypi.ngc.nvidia.com
Requirement already satisfied: openai==0.27.4 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 1)) (0.27.4)
Requirement already satisfied: flask==2.2.3 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 2)) (2.2.3)
Requirement already satisfied: flask_cors==3.0.10 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 3)) (3.0.10)
Requirement already satisfied: aiohttp in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (3.8.4)
Requirement already satisfied: requests>=2.20 in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (2.29.0)
Requirement already satisfied: tqdm in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (4.65.0)
Requirement already satisfied: Jinja2>=3.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (3.1.2)
Requirement already satisfied: click>=8.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (8.1.3)
Requirement already satisfied: Werkzeug>=2.2.2 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (2.3.4)
Requirement already satisfied: itsdangerous>=2.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (2.1.2)
Requirement already satisfied: Six in /usr/local/lib/python3.10/dist-packages (from flask_cors==3.0.10->-r ../../requirements.txt (line 3)) (1.16.0)
Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from Jinja2>=3.0->flask==2.2.3->-r ../../requirements.txt (line 2)) (2.1.2)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.26.15)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (2022.12.7)
Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (3.1.0)
Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (3.4)
Requirement already satisfied: yarl<2.0,>=1.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.9.2)
Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (4.0.2)
Requirement already satisfied: aiosignal>=1.1.2 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.3.1)
Requirement already satisfied: multidict<7.0,>=4.5 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (6.0.4)
Requirement already satisfied: frozenlist>=1.1.1 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.3.3)
Requirement already satisfied: attrs>=17.3.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (23.1.0)
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
WARNING: You are using pip version 21.2.4; however, version 23.1.2 is available.
You should consider upgrading via the '/usr/bin/python -m pip install --upgrade pip' command.
jq: error: syntax error, unexpected IDENT, expecting $end (Unix shell quoting issues?) at <top-level>, line 1:
. | ..llm      
jq: 1 compile error
jq: error: syntax error, unexpected IDENT, expecting $end (Unix shell quoting issues?) at <top-level>, line 1:
 . | ..env        
jq: 1 compile error
Loading model with type 
./run_llm.sh: line 24: [: =: unary operator expected
cannot find configuration for model 
Looking in indexes: http://pypi-cache/index, https://pypi.ngc.nvidia.com
Requirement already satisfied: openai==0.27.4 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 1)) (0.27.4)
Requirement already satisfied: flask==2.2.3 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 2)) (2.2.3)
Requirement already satisfied: flask_cors==3.0.10 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 3)) (3.0.10)
Requirement already satisfied: aiohttp in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (3.8.4)
Requirement already satisfied: tqdm in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (4.65.0)
Requirement already satisfied: requests>=2.20 in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (2.29.0)
Requirement already satisfied: Werkzeug>=2.2.2 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (2.3.4)
Requirement already satisfied: Jinja2>=3.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (3.1.2)
Requirement already satisfied: itsdangerous>=2.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (2.1.2)
Requirement already satisfied: click>=8.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (8.1.3)
Requirement already satisfied: Six in /usr/local/lib/python3.10/dist-packages (from flask_cors==3.0.10->-r ../../requirements.txt (line 3)) (1.16.0)
Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from Jinja2>=3.0->flask==2.2.3->-r ../../requirements.txt (line 2)) (2.1.2)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (2022.12.7)
Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (3.4)
Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (3.1.0)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.26.15)
Requirement already satisfied: multidict<7.0,>=4.5 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (6.0.4)
Requirement already satisfied: aiosignal>=1.1.2 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.3.1)
Requirement already satisfied: frozenlist>=1.1.1 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.3.3)
Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (4.0.2)
Requirement already satisfied: yarl<2.0,>=1.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.9.2)
Requirement already satisfied: attrs>=17.3.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (23.1.0)
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
WARNING: You are using pip version 21.2.4; however, version 23.1.2 is available.
You should consider upgrading via the '/usr/bin/python -m pip install --upgrade pip' command.
jq: error: syntax error, unexpected IDENT, expecting $end (Unix shell quoting issues?) at <top-level>, line 1:
. | ..llm      
jq: 1 compile error
jq: error: syntax error, unexpected IDENT, expecting $end (Unix shell quoting issues?) at <top-level>, line 1:
 . | ..env        
jq: 1 compile error
Loading model with type 
./run_llm.sh: line 24: [: =: unary operator expected
cannot find configuration for model 
Looking in indexes: http://pypi-cache/index, https://pypi.ngc.nvidia.com
Requirement already satisfied: openai==0.27.4 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 1)) (0.27.4)
Requirement already satisfied: flask==2.2.3 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 2)) (2.2.3)
Requirement already satisfied: flask_cors==3.0.10 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 3)) (3.0.10)
Requirement already satisfied: aiohttp in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (3.8.4)
Requirement already satisfied: requests>=2.20 in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (2.29.0)
Requirement already satisfied: tqdm in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (4.65.0)
Requirement already satisfied: click>=8.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (8.1.3)
Requirement already satisfied: Werkzeug>=2.2.2 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (2.3.4)
Requirement already satisfied: Jinja2>=3.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (3.1.2)
Requirement already satisfied: itsdangerous>=2.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (2.1.2)
Requirement already satisfied: Six in /usr/local/lib/python3.10/dist-packages (from flask_cors==3.0.10->-r ../../requirements.txt (line 3)) (1.16.0)
Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from Jinja2>=3.0->flask==2.2.3->-r ../../requirements.txt (line 2)) (2.1.2)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (2022.12.7)
Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (3.1.0)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.26.15)
Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (3.4)
Requirement already satisfied: attrs>=17.3.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (23.1.0)
Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (4.0.2)
Requirement already satisfied: yarl<2.0,>=1.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.9.2)
Requirement already satisfied: multidict<7.0,>=4.5 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (6.0.4)
Requirement already satisfied: aiosignal>=1.1.2 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.3.1)
Requirement already satisfied: frozenlist>=1.1.1 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.3.3)
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
WARNING: You are using pip version 21.2.4; however, version 23.1.2 is available.
You should consider upgrading via the '/usr/bin/python -m pip install --upgrade pip' command.
jq: error: t5/0 is not defined at <top-level>, line 1:
. | .flan-t5-large.llm          
jq: error: large/0 is not defined at <top-level>, line 1:
. | .flan-t5-large.llm             
jq: 2 compile errors
jq: error: t5/0 is not defined at <top-level>, line 1:
 . | .flan-t5-large.env            
jq: error: large/0 is not defined at <top-level>, line 1:
 . | .flan-t5-large.env               
jq: 2 compile errors
Loading model with type 
./run_llm.sh: line 24: [: =: unary operator expected
cannot find configuration for model 
Looking in indexes: http://pypi-cache/index, https://pypi.ngc.nvidia.com
Requirement already satisfied: openai==0.27.4 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 1)) (0.27.4)
Requirement already satisfied: flask==2.2.3 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 2)) (2.2.3)
Requirement already satisfied: flask_cors==3.0.10 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 3)) (3.0.10)
Requirement already satisfied: requests>=2.20 in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (2.29.0)
Requirement already satisfied: tqdm in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (4.65.0)
Requirement already satisfied: aiohttp in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (3.8.4)
Requirement already satisfied: itsdangerous>=2.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (2.1.2)
Requirement already satisfied: Jinja2>=3.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (3.1.2)
Requirement already satisfied: click>=8.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (8.1.3)
Requirement already satisfied: Werkzeug>=2.2.2 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (2.3.4)
Requirement already satisfied: Six in /usr/local/lib/python3.10/dist-packages (from flask_cors==3.0.10->-r ../../requirements.txt (line 3)) (1.16.0)
Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from Jinja2>=3.0->flask==2.2.3->-r ../../requirements.txt (line 2)) (2.1.2)
Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (3.4)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (2022.12.7)
Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (3.1.0)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.26.15)
Requirement already satisfied: frozenlist>=1.1.1 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.3.3)
Requirement already satisfied: yarl<2.0,>=1.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.9.2)
Requirement already satisfied: multidict<7.0,>=4.5 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (6.0.4)
Requirement already satisfied: attrs>=17.3.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (23.1.0)
Requirement already satisfied: aiosignal>=1.1.2 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.3.1)
Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (4.0.2)
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
WARNING: You are using pip version 21.2.4; however, version 23.1.2 is available.
You should consider upgrading via the '/usr/bin/python -m pip install --upgrade pip' command.
jq: error: t5/0 is not defined at <top-level>, line 1:
. | .flan-t5-large.llm          
jq: error: large/0 is not defined at <top-level>, line 1:
. | .flan-t5-large.llm             
jq: 2 compile errors
jq: error: t5/0 is not defined at <top-level>, line 1:
 . | .flan-t5-large.env            
jq: error: large/0 is not defined at <top-level>, line 1:
 . | .flan-t5-large.env               
jq: 2 compile errors
Loading model with type 
./run_llm.sh: line 24: [: =: unary operator expected
cannot find configuration for model 
Looking in indexes: http://pypi-cache/index, https://pypi.ngc.nvidia.com
Requirement already satisfied: openai==0.27.4 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 1)) (0.27.4)
Requirement already satisfied: flask==2.2.3 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 2)) (2.2.3)
Requirement already satisfied: flask_cors==3.0.10 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 3)) (3.0.10)
Requirement already satisfied: aiohttp in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (3.8.4)
Requirement already satisfied: requests>=2.20 in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (2.29.0)
Requirement already satisfied: tqdm in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (4.65.0)
Requirement already satisfied: Jinja2>=3.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (3.1.2)
Requirement already satisfied: click>=8.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (8.1.3)
Requirement already satisfied: itsdangerous>=2.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (2.1.2)
Requirement already satisfied: Werkzeug>=2.2.2 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (2.3.4)
Requirement already satisfied: Six in /usr/local/lib/python3.10/dist-packages (from flask_cors==3.0.10->-r ../../requirements.txt (line 3)) (1.16.0)
Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from Jinja2>=3.0->flask==2.2.3->-r ../../requirements.txt (line 2)) (2.1.2)
Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (3.4)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (2022.12.7)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.26.15)
Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (3.1.0)
Requirement already satisfied: aiosignal>=1.1.2 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.3.1)
Requirement already satisfied: multidict<7.0,>=4.5 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (6.0.4)
Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (4.0.2)
Requirement already satisfied: yarl<2.0,>=1.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.9.2)
Requirement already satisfied: frozenlist>=1.1.1 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.3.3)
Requirement already satisfied: attrs>=17.3.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (23.1.0)
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
WARNING: You are using pip version 21.2.4; however, version 23.1.2 is available.
You should consider upgrading via the '/usr/bin/python -m pip install --upgrade pip' command.
jq: error (at <stdin>:1): null (null) has no keys
Loading model with type t5
Looking in indexes: http://pypi-cache/index, https://pypi.ngc.nvidia.com
Requirement already satisfied: transformers==4.29.2 in /usr/local/lib/python3.10/dist-packages (from -r llm/wrapper/t5/requirements.txt (line 1)) (4.29.2)
Requirement already satisfied: sentencepiece==0.1.99 in /usr/local/lib/python3.10/dist-packages (from -r llm/wrapper/t5/requirements.txt (line 2)) (0.1.99)
Requirement already satisfied: accelerate==0.20.3 in /usr/local/lib/python3.10/dist-packages (from -r llm/wrapper/t5/requirements.txt (line 3)) (0.20.3)
Requirement already satisfied: tensorflow==2.12.0 in /usr/local/lib/python3.10/dist-packages (from -r llm/wrapper/t5/requirements.txt (line 4)) (2.12.0)
Requirement already satisfied: requests in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (2.29.0)
Requirement already satisfied: pyyaml>=5.1 in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (6.0)
Requirement already satisfied: filelock in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (3.12.0)
Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (23.1)
Requirement already satisfied: tqdm>=4.27 in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (4.65.0)
Requirement already satisfied: regex!=2019.12.17 in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (2023.5.5)
Requirement already satisfied: tokenizers!=0.11.3,<0.14,>=0.11.1 in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (0.13.3)
Requirement already satisfied: huggingface-hub<1.0,>=0.14.1 in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (0.15.1)
Requirement already satisfied: numpy>=1.17 in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (1.22.2)
Requirement already satisfied: torch>=1.6.0 in /usr/local/lib/python3.10/dist-packages (from accelerate==0.20.3->-r llm/wrapper/t5/requirements.txt (line 3)) (2.0.0)
Requirement already satisfied: psutil in /usr/local/lib/python3.10/dist-packages (from accelerate==0.20.3->-r llm/wrapper/t5/requirements.txt (line 3)) (5.9.4)
Requirement already satisfied: flatbuffers>=2.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (23.5.26)
Requirement already satisfied: jax>=0.3.15 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.4.13)
Requirement already satisfied: typing-extensions>=3.6.6 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (4.5.0)
Requirement already satisfied: setuptools in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (65.5.1)
Requirement already satisfied: opt-einsum>=2.3.2 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (3.3.0)
Requirement already satisfied: protobuf!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.20.3 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (3.20.3)
Requirement already satisfied: gast<=0.4.0,>=0.2.1 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.4.0)
Requirement already satisfied: keras<2.13,>=2.12.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (2.12.0)
Requirement already satisfied: astunparse>=1.6.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.6.3)
Requirement already satisfied: termcolor>=1.1.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (2.3.0)
Requirement already satisfied: six>=1.12.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.16.0)
Requirement already satisfied: google-pasta>=0.1.1 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.2.0)
Requirement already satisfied: libclang>=13.0.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (16.0.0)
Requirement already satisfied: absl-py>=1.0.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.4.0)
Requirement already satisfied: h5py>=2.9.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (3.9.0)
Requirement already satisfied: grpcio<2.0,>=1.24.3 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.54.2)
Requirement already satisfied: tensorflow-io-gcs-filesystem>=0.23.1 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.32.0)
Requirement already satisfied: wrapt<1.15,>=1.11.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.14.1)
Requirement already satisfied: tensorflow-estimator<2.13,>=2.12.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (2.12.0)
Requirement already satisfied: tensorboard<2.13,>=2.12 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (2.12.3)
Requirement already satisfied: wheel<1.0,>=0.23.0 in /usr/local/lib/python3.10/dist-packages (from astunparse>=1.6.0->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.40.0)
Requirement already satisfied: fsspec in /usr/local/lib/python3.10/dist-packages (from huggingface-hub<1.0,>=0.14.1->transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (2023.4.0)
Requirement already satisfied: scipy>=1.7 in /usr/local/lib/python3.10/dist-packages (from jax>=0.3.15->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.10.1)
Requirement already satisfied: ml-dtypes>=0.1.0 in /usr/local/lib/python3.10/dist-packages (from jax>=0.3.15->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.2.0)
Requirement already satisfied: google-auth<3,>=1.6.3 in /usr/local/lib/python3.10/dist-packages (from tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (2.18.1)
Requirement already satisfied: google-auth-oauthlib<1.1,>=0.5 in /usr/local/lib/python3.10/dist-packages (from tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.0.0)
Requirement already satisfied: werkzeug>=1.0.1 in /usr/local/lib/python3.10/dist-packages (from tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (2.3.4)
Requirement already satisfied: markdown>=2.6.8 in /usr/local/lib/python3.10/dist-packages (from tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (3.4.3)
Requirement already satisfied: tensorboard-data-server<0.8.0,>=0.7.0 in /usr/local/lib/python3.10/dist-packages (from tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.7.1)
Requirement already satisfied: urllib3<2.0 in /usr/local/lib/python3.10/dist-packages (from google-auth<3,>=1.6.3->tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.26.15)
Requirement already satisfied: cachetools<6.0,>=2.0.0 in /usr/local/lib/python3.10/dist-packages (from google-auth<3,>=1.6.3->tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (5.3.0)
Requirement already satisfied: rsa<5,>=3.1.4 in /usr/local/lib/python3.10/dist-packages (from google-auth<3,>=1.6.3->tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (4.9)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /usr/local/lib/python3.10/dist-packages (from google-auth<3,>=1.6.3->tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.3.0)
Requirement already satisfied: requests-oauthlib>=0.7.0 in /usr/local/lib/python3.10/dist-packages (from google-auth-oauthlib<1.1,>=0.5->tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.3.1)
Requirement already satisfied: pyasn1<0.6.0,>=0.4.6 in /usr/local/lib/python3.10/dist-packages (from pyasn1-modules>=0.2.1->google-auth<3,>=1.6.3->tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.5.0)
Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests->transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (3.1.0)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests->transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (2022.12.7)
Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests->transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (3.4)
Requirement already satisfied: oauthlib>=3.0.0 in /usr/local/lib/python3.10/dist-packages (from requests-oauthlib>=0.7.0->google-auth-oauthlib<1.1,>=0.5->tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (3.2.2)
Requirement already satisfied: sympy in /usr/local/lib/python3.10/dist-packages (from torch>=1.6.0->accelerate==0.20.3->-r llm/wrapper/t5/requirements.txt (line 3)) (1.12)
Requirement already satisfied: networkx in /usr/local/lib/python3.10/dist-packages (from torch>=1.6.0->accelerate==0.20.3->-r llm/wrapper/t5/requirements.txt (line 3)) (2.6.3)
Requirement already satisfied: jinja2 in /usr/local/lib/python3.10/dist-packages (from torch>=1.6.0->accelerate==0.20.3->-r llm/wrapper/t5/requirements.txt (line 3)) (3.1.2)
Requirement already satisfied: MarkupSafe>=2.1.1 in /usr/local/lib/python3.10/dist-packages (from werkzeug>=1.0.1->tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (2.1.2)
Requirement already satisfied: mpmath>=0.19 in /usr/local/lib/python3.10/dist-packages (from sympy->torch>=1.6.0->accelerate==0.20.3->-r llm/wrapper/t5/requirements.txt (line 3)) (1.3.0)
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
WARNING: You are using pip version 21.2.4; however, version 23.1.2 is available.
You should consider upgrading via the '/usr/bin/python -m pip install --upgrade pip' command.
starting t5 with huggingface_model=google/flan-t5-large
2023-07-04 15:36:31.109488: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
 * Serving Flask app 'llm.http_api'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
Looking in indexes: http://pypi-cache/index, https://pypi.ngc.nvidia.com
Requirement already satisfied: openai==0.27.4 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 1)) (0.27.4)
Requirement already satisfied: flask==2.2.3 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 2)) (2.2.3)
Requirement already satisfied: flask_cors==3.0.10 in /usr/local/lib/python3.10/dist-packages (from -r ../../requirements.txt (line 3)) (3.0.10)
Requirement already satisfied: aiohttp in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (3.8.4)
Requirement already satisfied: tqdm in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (4.65.0)
Requirement already satisfied: requests>=2.20 in /usr/local/lib/python3.10/dist-packages (from openai==0.27.4->-r ../../requirements.txt (line 1)) (2.29.0)
Requirement already satisfied: itsdangerous>=2.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (2.1.2)
Requirement already satisfied: Jinja2>=3.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (3.1.2)
Requirement already satisfied: Werkzeug>=2.2.2 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (2.3.4)
Requirement already satisfied: click>=8.0 in /usr/local/lib/python3.10/dist-packages (from flask==2.2.3->-r ../../requirements.txt (line 2)) (8.1.3)
Requirement already satisfied: Six in /usr/local/lib/python3.10/dist-packages (from flask_cors==3.0.10->-r ../../requirements.txt (line 3)) (1.16.0)
Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from Jinja2>=3.0->flask==2.2.3->-r ../../requirements.txt (line 2)) (2.1.2)
Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (3.4)
Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (3.1.0)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.26.15)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests>=2.20->openai==0.27.4->-r ../../requirements.txt (line 1)) (2022.12.7)
Requirement already satisfied: yarl<2.0,>=1.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.9.2)
Requirement already satisfied: multidict<7.0,>=4.5 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (6.0.4)
Requirement already satisfied: aiosignal>=1.1.2 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.3.1)
Requirement already satisfied: frozenlist>=1.1.1 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (1.3.3)
Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (4.0.2)
Requirement already satisfied: attrs>=17.3.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->openai==0.27.4->-r ../../requirements.txt (line 1)) (23.1.0)
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
WARNING: You are using pip version 21.2.4; however, version 23.1.2 is available.
You should consider upgrading via the '/usr/bin/python -m pip install --upgrade pip' command.
jq: error (at <stdin>:1): null (null) has no keys
Loading model with type t5
Looking in indexes: http://pypi-cache/index, https://pypi.ngc.nvidia.com
Requirement already satisfied: transformers==4.29.2 in /usr/local/lib/python3.10/dist-packages (from -r llm/wrapper/t5/requirements.txt (line 1)) (4.29.2)
Requirement already satisfied: sentencepiece==0.1.99 in /usr/local/lib/python3.10/dist-packages (from -r llm/wrapper/t5/requirements.txt (line 2)) (0.1.99)
Requirement already satisfied: accelerate==0.20.3 in /usr/local/lib/python3.10/dist-packages (from -r llm/wrapper/t5/requirements.txt (line 3)) (0.20.3)
Requirement already satisfied: tensorflow==2.12.0 in /usr/local/lib/python3.10/dist-packages (from -r llm/wrapper/t5/requirements.txt (line 4)) (2.12.0)
Requirement already satisfied: tqdm>=4.27 in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (4.65.0)
Requirement already satisfied: pyyaml>=5.1 in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (6.0)
Requirement already satisfied: regex!=2019.12.17 in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (2023.5.5)
Requirement already satisfied: huggingface-hub<1.0,>=0.14.1 in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (0.15.1)
Requirement already satisfied: filelock in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (3.12.0)
Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (23.1)
Requirement already satisfied: tokenizers!=0.11.3,<0.14,>=0.11.1 in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (0.13.3)
Requirement already satisfied: numpy>=1.17 in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (1.22.2)
Requirement already satisfied: requests in /usr/local/lib/python3.10/dist-packages (from transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (2.29.0)
Requirement already satisfied: psutil in /usr/local/lib/python3.10/dist-packages (from accelerate==0.20.3->-r llm/wrapper/t5/requirements.txt (line 3)) (5.9.4)
Requirement already satisfied: torch>=1.6.0 in /usr/local/lib/python3.10/dist-packages (from accelerate==0.20.3->-r llm/wrapper/t5/requirements.txt (line 3)) (2.0.0)
Requirement already satisfied: tensorboard<2.13,>=2.12 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (2.12.3)
Requirement already satisfied: google-pasta>=0.1.1 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.2.0)
Requirement already satisfied: libclang>=13.0.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (16.0.0)
Requirement already satisfied: termcolor>=1.1.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (2.3.0)
Requirement already satisfied: absl-py>=1.0.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.4.0)
Requirement already satisfied: keras<2.13,>=2.12.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (2.12.0)
Requirement already satisfied: jax>=0.3.15 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.4.13)
Requirement already satisfied: grpcio<2.0,>=1.24.3 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.54.2)
Requirement already satisfied: tensorflow-io-gcs-filesystem>=0.23.1 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.32.0)
Requirement already satisfied: flatbuffers>=2.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (23.5.26)
Requirement already satisfied: six>=1.12.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.16.0)
Requirement already satisfied: tensorflow-estimator<2.13,>=2.12.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (2.12.0)
Requirement already satisfied: setuptools in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (65.5.1)
Requirement already satisfied: h5py>=2.9.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (3.9.0)
Requirement already satisfied: opt-einsum>=2.3.2 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (3.3.0)
Requirement already satisfied: typing-extensions>=3.6.6 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (4.5.0)
Requirement already satisfied: astunparse>=1.6.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.6.3)
Requirement already satisfied: wrapt<1.15,>=1.11.0 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.14.1)
Requirement already satisfied: protobuf!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.20.3 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (3.20.3)
Requirement already satisfied: gast<=0.4.0,>=0.2.1 in /usr/local/lib/python3.10/dist-packages (from tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.4.0)
Requirement already satisfied: wheel<1.0,>=0.23.0 in /usr/local/lib/python3.10/dist-packages (from astunparse>=1.6.0->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.40.0)
Requirement already satisfied: fsspec in /usr/local/lib/python3.10/dist-packages (from huggingface-hub<1.0,>=0.14.1->transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (2023.4.0)
Requirement already satisfied: scipy>=1.7 in /usr/local/lib/python3.10/dist-packages (from jax>=0.3.15->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.10.1)
Requirement already satisfied: ml-dtypes>=0.1.0 in /usr/local/lib/python3.10/dist-packages (from jax>=0.3.15->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.2.0)
Requirement already satisfied: werkzeug>=1.0.1 in /usr/local/lib/python3.10/dist-packages (from tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (2.3.4)
Requirement already satisfied: google-auth-oauthlib<1.1,>=0.5 in /usr/local/lib/python3.10/dist-packages (from tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.0.0)
Requirement already satisfied: google-auth<3,>=1.6.3 in /usr/local/lib/python3.10/dist-packages (from tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (2.18.1)
Requirement already satisfied: tensorboard-data-server<0.8.0,>=0.7.0 in /usr/local/lib/python3.10/dist-packages (from tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.7.1)
Requirement already satisfied: markdown>=2.6.8 in /usr/local/lib/python3.10/dist-packages (from tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (3.4.3)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /usr/local/lib/python3.10/dist-packages (from google-auth<3,>=1.6.3->tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.3.0)
Requirement already satisfied: urllib3<2.0 in /usr/local/lib/python3.10/dist-packages (from google-auth<3,>=1.6.3->tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.26.15)
Requirement already satisfied: cachetools<6.0,>=2.0.0 in /usr/local/lib/python3.10/dist-packages (from google-auth<3,>=1.6.3->tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (5.3.0)
Requirement already satisfied: rsa<5,>=3.1.4 in /usr/local/lib/python3.10/dist-packages (from google-auth<3,>=1.6.3->tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (4.9)
Requirement already satisfied: requests-oauthlib>=0.7.0 in /usr/local/lib/python3.10/dist-packages (from google-auth-oauthlib<1.1,>=0.5->tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (1.3.1)
Requirement already satisfied: pyasn1<0.6.0,>=0.4.6 in /usr/local/lib/python3.10/dist-packages (from pyasn1-modules>=0.2.1->google-auth<3,>=1.6.3->tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (0.5.0)
Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests->transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (3.1.0)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests->transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (2022.12.7)
Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests->transformers==4.29.2->-r llm/wrapper/t5/requirements.txt (line 1)) (3.4)
Requirement already satisfied: oauthlib>=3.0.0 in /usr/local/lib/python3.10/dist-packages (from requests-oauthlib>=0.7.0->google-auth-oauthlib<1.1,>=0.5->tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (3.2.2)
Requirement already satisfied: sympy in /usr/local/lib/python3.10/dist-packages (from torch>=1.6.0->accelerate==0.20.3->-r llm/wrapper/t5/requirements.txt (line 3)) (1.12)
Requirement already satisfied: networkx in /usr/local/lib/python3.10/dist-packages (from torch>=1.6.0->accelerate==0.20.3->-r llm/wrapper/t5/requirements.txt (line 3)) (2.6.3)
Requirement already satisfied: jinja2 in /usr/local/lib/python3.10/dist-packages (from torch>=1.6.0->accelerate==0.20.3->-r llm/wrapper/t5/requirements.txt (line 3)) (3.1.2)
Requirement already satisfied: MarkupSafe>=2.1.1 in /usr/local/lib/python3.10/dist-packages (from werkzeug>=1.0.1->tensorboard<2.13,>=2.12->tensorflow==2.12.0->-r llm/wrapper/t5/requirements.txt (line 4)) (2.1.2)
Requirement already satisfied: mpmath>=0.19 in /usr/local/lib/python3.10/dist-packages (from sympy->torch>=1.6.0->accelerate==0.20.3->-r llm/wrapper/t5/requirements.txt (line 3)) (1.3.0)
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
WARNING: You are using pip version 21.2.4; however, version 23.1.2 is available.
You should consider upgrading via the '/usr/bin/python -m pip install --upgrade pip' command.
starting t5 with huggingface_model=google/flan-t5-large
2023-07-04 15:38:32.184007: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
 * Serving Flask app 'llm.http_api'
 * Debug mode: on
2023-07-04 15:40:28.106132: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1956] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
